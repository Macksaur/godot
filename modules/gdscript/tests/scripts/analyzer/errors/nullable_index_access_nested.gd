class B:
    var bar

class A:
    var foo: B?

func test():
    var foobar: Array[A?] = [A.new()]
    foobar[0].bar