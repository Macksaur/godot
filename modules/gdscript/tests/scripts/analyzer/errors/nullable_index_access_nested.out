GDTEST_ANALYZER_ERROR
Potentially unsafe to access nullable "foobar[0]" with attribute "bar".
